icon.svg

```diff
@@ -1,35 +1,57 @@
-c//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128">
-	<def>
-		<path id="bracket" d="M50,123 h-8 c-20,0 -24,-6 -20,-30 c2,-15 2,-18 -10,-20 v-10 c12,-2 12,-5 10,-20 c-4,-24 0,-30 20,-30 h8 v9 h-6 c-13,0 -13,2 -11,20 c2,20, 1,23 -10,25.5 v1 c11,2.5 12,5.5 10,25.5 c-2,18 -2,20 11,20 h6z"
-		/>
-		<g id="pet">
-			<path fill="#D500DF" d="M0,0 C0,0 -10,-10 -10,-20 C-10,-26 -6,-30 0,-30 C6,-30 10,-26 10,-20 C10,-10 0,0 0,0" />
-			<path fill="#FFFFFF" d="M0,0 C0,0  -7,-10  -7,-18 C -7,-23 -4,-26 0,-26 C4,-26  7,-23  7,-18 C 7,-10 0,0 0,0" />
-			<path fill="#D500DF" d="M0,0 C-2,0 -2,-20 0,-20 C2,-20, 2,0 0,0" />
-		</g>
-		<g id="main">
-			<use xlink:href="#bracket" transform="translate(-6)" />
-			<use transform="matrix(-1 0 0 1 118 0)" xlink:href="#bracket" />
-			<g text-anchor="middle" font-size="16" font-weight="bold" font-family="monospace">
-				<text x="56" y="39">js</text>
-				<text x="56" y="56">json</text>
-				<text x="56" y="73">html</text>
-				<text x="56" y="89">css</text>
-				<text x="56" y="105">scss</text>
-			</g>
-		</g>
-	</def>
-	<rect x="0" y="0" width="128" height="128" rx="16" ry="16" fill="#e8e030" />
-	<use xlink:href="#main" />
-	<g transform="matrix(0.75 0 0 0.75 100 30)">
-		<use transform="rotate(0)" xlink:href="#pet" />
-		<use transform="rotate(72)" xlink:href="#pet" />
-		<use transform="rotate(144)" xlink:href="#pet" />
-		<use transform="rotate(216)" xlink:href="#pet" />
-		<use transform="rotate(288)" xlink:href="#pet" />
-		<circle fill="#D500DF" cx="0" cy="0" r="10" />
-	</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#E8E030;}
+	.st1{fill:#D91CE8;}
+	.st2{fill:#E5C230;}
+	.st3{fill:#D500DF;}
+	.st4{fill:#FFFFFF;}
+</style>
+<g id="Circle">
+	<circle class="st0" cx="64" cy="64" r="64"/>
+	<path class="st1" d="M64,0C28.7,0,0,28.7,0,64s28.7,64,64,64s64-28.7,64-64S99.3,0,64,0z M64,122.5C31.7,122.5,5.5,96.3,5.5,64
+		S31.7,5.5,64,5.5s58.5,26.2,58.5,58.5S96.3,122.5,64,122.5z M54.8,127.4L54.8,127.4"/>
+	<path class="st2" d="M125.7,80.9L101.3,48l-10,3.1L75.8,29.7L57.2,54.8l-31.1,7.9L41,87l-4.4,6.8l20.6,33.8
+		c2.3,0.2,4.5,0.4,6.9,0.4C93.5,128,118.3,108,125.7,80.9z"/>
+	<path class="st3" d="M125.7,80.9l-4.3-5.8c-5.2,27-28.9,47.4-57.4,47.4c-3.6,0-7.1-0.3-10.6-1l3.7,6.1c2.3,0.2,4.5,0.4,6.9,0.4
+		C93.5,128,118.3,108,125.7,80.9z"/>
+</g>
+<g id="Flower">
+	<g id="Pedals">
+		<g id="pet">
+			<path class="st3" d="M64,64.1c0,0-13.7-13.6-13.7-27.3c0-8.2,5.5-13.6,13.7-13.6s13.7,5.5,13.7,13.7C77.7,50.5,64,64.1,64,64.1"
```

Comment:
Each of the petals is still of by 0.1px. (64.1 vs 64) The centre circle is now correct though.